class.json-api-endpoints.php

```diff
@@ -295,11 +295,11 @@ function input( $return_default_values = true, $cast_and_filter = true ) {
 
 
 	protected function get_secure_body( $secure_key ) {
-		$response =  Jetpack_Client::wpcom_json_api_request_as_blog( 
-			sprintf( '/sites/%d/secure-request', Jetpack_Options::get_option('id' ) ), 
-			'1.1', 
-			array( 'method' => 'POST' ), 
-			array( 'secure_key' => $secure_key ) 
+		$response =  Jetpack_Client::wpcom_json_api_request_as_blog(
+			sprintf( '/sites/%d/secure-request', Jetpack_Options::get_option('id' ) ),
+			'1.1',
+			array( 'method' => 'POST' ),
+			array( 'secure_key' => $secure_key )
 		);
 		if ( 200 !== $response['response']['code'] ) {
```

Comment:
Yeah I know, I was just seeing that this could be updated too, but it's not necessary for the scope of this PR.